site/getstarted/install.md

```diff
@@ -1,6 +1,6 @@
 # Install
 
-The latest version for Cats Effect 3 is `3.2.0`, which supports Cats Effect 3 and is cross built for Scala 2.12, 2.13, and 3.0.
+The latest version for Cats Effect 3 is `3.2.4`, which supports Cats Effect 3 and is cross built for Scala 2.12, 2.13, and 3.0.
```

Comment:
Actually :) https://github.com/typelevel/cats-effect/releases/tag/v3.3.5

```suggestion
The latest version for Cats Effect 3 is `3.3.5`, which supports Cats Effect 3 and is cross built for Scala 2.12, 2.13, and 3.0.
```

We could set this automatically with mdoc btw.

Reply:
Woops sorry, ignore me

Reply:
Wait, how?

Reply:
Here's how I do it in the sbt-typelevel-site plugin:
https://github.com/typelevel/sbt-typelevel/blob/3e343113edd5ae1153505efd7feff57579b51cfa/site/src/main/scala/org/typelevel/sbt/TypelevelSitePlugin.scala#L64-L70

https://github.com/typelevel/sbt-typelevel/blob/3e343113edd5ae1153505efd7feff57579b51cfa/docs/index.md?plain=1#L14

Actually, I can extract those settings out into a `TypelevelMdocPlugin` so you can get those variables for free without buying into the rest of the sbt-typelevel-site stuff.